src/components/Calculator.js

```diff
@@ -1,54 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      operation: null,
-      next: null,
-    };
-    this.updateCal = this.updateCal.bind(this);
-  }
+export default function Calculator() {
```

Comment:
## Optional

- [ ] Good Job doing those changes. I recommend using arrow functions as good practice for your next projects.